### DISCUSSION TITLE
Why can't we use pnpm but we can use yarn?

### DISCUSSION THREAD

**Discussion by misogihagi** [community/community#160577] [Actions]
yarn is pre-installed, but pnpm is not.
pnpm is becoming increasingly popular. Why is pnpm not available by default?

**Comment by kishorekumar-anchala** [comment]
Hi @misogihagi , Good catch !. 

   npm and yarn are very popular and have more customers are using them when compare to pnpm.  In recent time pnpm getting more users . But , most of the users are unable to adapt it as compare to npm and yarn (i believe).

**Comment by susmitamane** [comment]
Hi @misogihagi ,

- Yarn is often preferred because it has better compatibility with existing tooling, broader ecosystem support, and is preinstalled on many platforms.
- Its seamless integration with package managers and hosting platforms makes it a preferred choice over pnpm.
- pnpm's symlink-based dependency management can introduce issues in some setups, making it less universally usable.

Thank you.
